Packs/Slack/Integrations/SlackV3/SlackV3_test.py

```diff
@@ -5232,29 +5257,23 @@ def test_get_war_room_url_with_xsiam_from_alert_war_room(self, mocker):
         assert get_war_room_url(url) == expected_war_room_url
 
 
-def test_send_file_deprecated_endpoint(mocker, requests_mock):
+def test_send_file_api_exception(mocker):
     """
     Given:
-        - Slack client of newaly created app (May 8,2024).
+        - Slack API non-okay response.
     When:
-        - calling the slack_send_file
+        - Calling the slack_send_file
     Then:
-        - assert readable exception is raised.
+        - Assert readable exception is raised.
     """
     from SlackV3 import slack_send_file
     # Set
     mocker.patch.object(demisto, 'args', return_value={})
     mocker.patch.object(demisto, 'investigation', return_value={'id': '999'})
-    mocker.patch.object(demisto, 'getIntegrationContext', side_effect=get_integration_context)
-    mocker.patch.object(demisto, 'setIntegrationContext', side_effect=set_integration_context)
     mocker.patch('SlackV3.slack_send_request', side_effect=SlackApiError('The request to the Slack API failed. (url: https://slack.com/api/files.upload)',
-                                                                         {'ok': False, 'error': 'method_deprecated'}))
+                                                                         {'ok': False, 'error': 'service_unavailable'}))
```

Comment:
why is this changed?

Reply:
I only changed it since we no longer have deprecated methods in our SlackV3 integration.

Other than that, there is no functional reason, so I will put back the original error message if you would like.